src/inventory.cpp

```diff
@@ -680,7 +680,11 @@ ItemStack InventoryList::removeItem(const ItemStack &item)
 	for (auto i = m_items.rbegin(); i != m_items.rend(); ++i) {
 		if (i->name == item.name) {
 			u32 still_to_remove = item.count - removed.count;
-			removed.addItem(i->takeItem(still_to_remove), m_itemdef);
+			ItemStack leftover = removed.addItem(i->takeItem(still_to_remove),
+					m_itemdef);
```

Comment:
Continuation lines only require one extra tab of indentation

Reply:
But two are OK too

Reply:
One is what our code-style demands, though...